pkg/webui/console/containers/device-title-section/index.js

```diff
@@ -58,19 +59,39 @@ const DeviceTitleSection = props => {
   const uplinkFrameCount = useSelector(state =>
     selectDeviceDerivedUplinkFrameCount(state, appId, devId),
   )
-  const downlinkFrameCount = useSelector(state =>
-    selectDeviceDerivedDownlinkFrameCount(state, appId, devId),
+  const downlinkAppFrameCount = useSelector(state =>
+    selectDeviceDerivedAppDownlinkFrameCount(state, appId, devId),
+  )
+  const downlinkNwkFrameCount = useSelector(state =>
+    selectDeviceDerivedNwkDownlinkFrameCount(state, appId, devId),
   )
   const lastSeen = useSelector(state => selectDeviceLastSeen(state, appId, devId))
   const showLastSeen = Boolean(lastSeen)
   const showUplinkCount = typeof uplinkFrameCount === 'number'
-  const showDownlinkCount = typeof downlinkFrameCount === 'number'
+  const showAppDownlinkCount = typeof downlinkAppFrameCount === 'number'
+  const showNwkDownlinkCount = typeof downlinkNwkFrameCount === 'number'
   const notAvailableElem = <Message content={sharedMessages.notAvailable} />
+  const downlinkValue =
+    showAppDownlinkCount && showNwkDownlinkCount
+      ? `${downlinkAppFrameCount} (App) / ${downlinkNwkFrameCount} (Nwk)`
+      : showAppDownlinkCount
+        ? `${downlinkAppFrameCount} (App)`
+        : showNwkDownlinkCount
+          ? `${downlinkNwkFrameCount} (Nwk)`
+          : notAvailableElem
```

Comment:
Please wrap the numbers in `<FormattedNumber />` otherwise they will display unformatted (see your screenshot).